package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vite-plugin-node-polyfills",
   "type": "module",
-  "version": "0.23.0",
+  "version": "0.23.1",
```

Comment:
I bump package versions after merging into the `main` branch. Please remove this change.